packages/styles/icon-button.css

```diff
@@ -1,46 +1,41 @@
 :root {
+  --icon-button-content-color-light-disabled: #ccc;
+  --icon-button-background-color-light: #dadada;
+  --icon-button-background-color-light-active: #c2c2c2;
+  --icon-button-outline-color-light: var(--icon-button-background-color-light);
+
+  --icon-button-content-color-dark-disabled: #fff 40%;
+  --icon-button-background-color-dark: var(--accent-medium);
+  --icon-button-background-color-dark-disabled: var(--accent-medium);
+  --icon-button-background-color-dark-active: var(--accent-dark);
+  --icon-button-outline-color-dark: var(--icon-button-background-color-dark);
+
+  /* Primary */
+  --icon-button-primary-color: var(--button-text-color-light);
   --icon-button-background-color-primary: var(
     --button-background-color-primary
   );
-  --icon-button-background-color-primary-disabled: var(
-    --button-background-color-primary-disabled
-  );
   --icon-button-background-color-primary-active: var(
     --button-background-color-primary-active
   );
   --icon-button-outline-color-primary: var(--button-outline-color-primary);
 
+  --icon-button-large-size: 36px;
+
+  /* Secondary */
+  --icon-button-secondary-color: var(--button-text-color-dark);
   --icon-button-background-color-secondary: var(
     --button-background-color-secondary
   );
-  --icon-button-background-color-secondary-disabled: var(
-    --button-background-color-secondary-disabled
-  );
   --icon-button-background-color-secondary-active: var(
     --button-background-color-secondary-active
   );
   --icon-button-outline-color-secondary: var(--button-outline-color-secondary);
   --icon-button-border-color-secondary: var(--gray-40);
 
+  /* Error */
   --icon-button-background-color-error: var(--button-background-color-error);
-  --icon-button-background-color-error-disabled: var(
```

Comment:
nding
This can probably be omitted, but we can't remove it without deprecation first (see related comment).